### PR TITLE
relativistic mass first commit

### PR DESCRIPTION
**Pull request by aokiyoshi**
Added law for relativistic mass

### CODE REVIEW

File: symplyphysics/laws/relativistic/relativistic_mass.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from sympy import Eq, solve, sqrt
+from sympy.physics.units import speed_of_light as c
```
Comment (blackyblack) [inline symplyphysics/laws/relativistic/relativistic_mass.py:2]: Don't rename to `c`

File: test/relativistic/relativistic_mass_test.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    v = Quantity(2_000_000 * (units.meter / units.second))
+    Args = namedtuple("Args", ["m", "v"])
+    return Args(m=m, v=v)
+
+
+def test_basic_mass(test_args):
+    result = relativistic_mass.calculate_relativistic_mass(
+        test_args.m, test_args.v)
+    result_mass = convert_to(result, units.kilogram).evalf(4)
+    assert result_mass == approx(6.04, 0.01)
```
Comment (blackyblack) [inline test/relativistic/relativistic_mass_test.py:24]: Where did you get this value. It is preferable to use some value from a book or exercise, to verify the correctness of the law.

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/relativistic/relativistic_mass.py (hunk 1 above)
Comment (aokiyoshi) [reply symplyphysics/laws/relativistic/relativistic_mass.py:2]: exact same thing was done in other file and it was ok (energy_is_mass.py)

File: test/relativistic/relativistic_mass_test.py (hunk 2 above)
Comment (aokiyoshi) [reply test/relativistic/relativistic_mass_test.py:24]: I calculated using calculator

File: symplyphysics/laws/relativistic/relativistic_mass.py (hunk 1 above)
Comment (blackyblack) [reply symplyphysics/laws/relativistic/relativistic_mass.py:2]: Thanks. I will fix that file as well.

**Pull request merged**